app/javascript/packs/application.js

```diff
@@ -18,9 +18,9 @@ window.ajax_events = ajax_events;
 // vendor libraries
 import 'javascripts/jquery.easyModal';
 
-// marked (markdown support)
-import marked from 'marked';
-window.marked = marked;
+// Markdown support (using marked and DOMpurify)
+import safe_marked from 'javascripts/safe_marked';
```

Comment:
Do we want to move towards a model where we import js libraries only in the files we need them instead of setting things globally on `window`?

If so, this might be a good opportunity to start doing that for this function at least.

I'm happy to make those changes if you want.

Reply:
Yeah sure, please go ahead and make those changes!

Reply:
This is actually going to require a bit more configuration that is out of scope for this PR. I'll do it separately once this is pulled in.